grails-app/views/species/tabs/_classification.gsp

```diff
@@ -48,7 +48,7 @@
                 </a>
 
                 <a
-                    href="${createLink(controller: 'species', action: 'search')}?q=${'rkid_' + tc.taxonConcept.rankString + ':' + tc.taxonConcept.guid}"
+                    href="${createLink(mapping: 'search')}?q=${tc.taxonConcept.rankString + 'ID_s:' + tc.taxonConcept.guid}""
```

Comment:
Are you sure that this line has to end with 2x double quotes?

Reply:
ffs